tests/acceptance/addon-generate-test.js

```diff
@@ -143,4 +143,10 @@ describe('Acceptance: ember generate in-addon', function () {
     await generateInAddon(['server']);
     expect(file('server/index.js')).to.exist;
   });
+
+  it('successfully generates the default blueprint for scoped addons', async function () {
+    await initAddon('@foo/bar');
```

Comment:
should this use the `generateInAddon` helper the other tests use?

Should there also be a test in `generate-test.js` for the non-addon case?

Reply:
> should this use the generateInAddon helper the other tests use?

`generateInAddon` currently uses a fixed addon name:
https://github.com/ember-cli/ember-cli/blob/61b839456af26d35bd9546e6932b85fdf94fcff1/tests/acceptance/addon-generate-test.js#L48

and uses `initAddon` internally.

I could make the addon name configurable, but to avoid having to edit the other tests, I would have to use `await generateInAddon(['@foo/bar'], '@foo/bar');`, which I find less readable than having the separate `initAddon` + `ember` calls. `generateInAddon` also only runs a single `ember` command.

> Should there also be a test in generate-test.js for the non-addon case?

I'll look into it.

Reply:
> I could make the addon name configurable

It seems the util already looks at the second argument.